interfaces/builtin/udisks2.go

```diff
@@ -217,18 +226,20 @@ socket AF_NETLINK - NETLINK_KOBJECT_UEVENT
 const udisks2PermanentSlotDBus = `
 <policy user="root">
     <allow own="org.freedesktop.UDisks2"/>
-    <allow send_destination="org.freedesktop.UDisks2"/>
 </policy>
 
 <policy context="default">
-    <allow send_destination="org.freedesktop.UDisks2" send_interface="org.freedesktop.DBus.Introspectable" />
+    <allow send_destination="org.freedesktop.UDisks2"/>
 </policy>
 `
 
 const udisks2ConnectedPlugDBus = `
 <policy context="default">
     <deny own="org.freedesktop.UDisks2"/>
-    <deny send_destination="org.freedesktop.UDisks2"/>
+</policy>
+
+<policy context="default">
+    <allow send_destination="org.freedesktop.UDisks2"/>
 </policy>
```

Comment:
Can this be under a single `<policy>` node?

Reply:
After thinking about your other comment, I feel there is something fishy behind this. Yes, it should be only one node. But I do not understand anymore the point of this file. So I am going to do some testing.

Reply:
I think this one did not make sense at all, so I removed it.